scripts/alter-package-json.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+
+/**
+ modify mozilla/shield-studies-addon-template/package.json in place
+ as part of `shield init`
+*/
+
+var jsonPath  = process.argv[2];
+var dir = process.argv[3];
+
+console.log(jsonPath, dir);
+var p = require(jsonPath);
+
+var oldName = 'shield-studies-addon-template';
+var newName = ['shield-study-',''][~~dir.startsWith('shield-study-')]+dir;
```

Comment:
I had to read this a few times to get the "gist" of it. Not sure if we can simplify or refactor this slightly.

This seems to give me similar results on my basic JSBin testing:
```js
const newName = dir.startsWith('shield-study-') ? dir : `shield-study-${dir}`;
```